server/src/HostInfoCache.cc

```diff
@@ -128,6 +128,21 @@ bool HostInfoCache::getName(
 	return found;
 }
 
+void HostInfoCache::registerAdHoc(const LocalHostIdType &idInServer,
+                                  const string &name, Element &elem)
+{
+	ServerHostDef svHostDef;
```

Comment:
svHostDef.id isn't initialized.
Although it's not used in the current implementation, it may cause problems in the future.